src/NewField.cpp

```diff
@@ -100,6 +101,11 @@ static void apply_render_info_to_layer(Actor* layer,
 	}
 }
 
+bool operator<(NewField::field_draw_entry const& rhs, NewField::field_draw_entry const& lhs)
```

Comment:
The < operator exists only so that I can use std::sort on a container of entries.  Defining the other operators would be a waste of time because they would be unused.